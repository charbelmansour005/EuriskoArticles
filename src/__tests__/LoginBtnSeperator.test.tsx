import React from 'react'
import renderer from 'react-test-renderer'
import LoginBtnSeperator from '../components/LoginBtnSepetator/LoginBtnSepetator'

describe('LoginBtnSeperator Component', () =>
  test('renders correctly', async () => {
    const tree = renderer.create(<LoginBtnSeperator />).toJSON()
    expect(tree).toMatchSnapshot()
  }))
