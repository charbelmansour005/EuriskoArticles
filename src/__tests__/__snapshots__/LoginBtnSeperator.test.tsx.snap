// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginBtnSeperator Component renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "justifyContent": "space-around",
      "marginTop": 25,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "black",
        "height": 1.2,
        "width": "35%",
      }
    }
  />
  <View>
    <Text
      style={
        Object {
          "color": "black",
          "textAlign": "center",
          "width": 50,
        }
      }
    >
      Or
    </Text>
  </View>
  <View
    style={
      Object {
        "backgroundColor": "black",
        "height": 1.2,
        "width": "35%",
      }
    }
  />
</View>
`;

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "justifyContent": "space-around",
      "marginTop": 25,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "black",
        "height": 1.2,
        "width": "35%",
      }
    }
  />
  <View>
    <Text
      style={
        Object {
          "color": "black",
          "textAlign": "center",
          "width": 50,
        }
      }
    >
      Or
    </Text>
  </View>
  <View
    style={
      Object {
        "backgroundColor": "black",
        "height": 1.2,
        "width": "35%",
      }
    }
  />
</View>
`;
